src/utils/bootstrap5-form-validation-renderer.ts

```diff
@@ -4,11 +4,11 @@ import {
 } from "aurelia-validation";
 
 /**
- * Form validation renderer for bootstrap 4
+ * Form validation renderer for bootstrap 5 (works for 4)
```

Comment:
Should have probably just removed the version number. I mean, it's not magically going to stop working for 4 now. ... But insignificant enough for me to accept anyway.